roles/matrix-base/templates/static-files/well-known/matrix-client.j2

```diff
@@ -26,12 +26,27 @@
 		"preferredDomain": {{ matrix_client_element_jitsi_preferredDomain|to_json }}
 	}
 	{% endif %}
-	{% if not matrix_client_element_e2ee_default %},
-	"io.element.e2ee": {
-		"default": false
+	,
+        "io.element.e2ee": {
+        	{% if not matrix_client_element_e2ee_default %}
+		   "default": false
```

Comment:
Instead of having `if` statements, we can simplify this by just doing:

```yaml
"default": {{ matrix_client_element_e2ee_default|to_json }}
```

Reply:
Hi, thanks for the quick answer and review.
I could push the requested changes, but I am not sure, answering here just commits your proposed changes. This feature of github is new to me.

Reply:
a I guess I need to resolve the conversation as soon as I have pushed the requested changes...